src/components/DropDetails.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { ReactComponent as RowDown } from '../assets/images/chevron-up.svg';
+
+export const DropDetails = props => {
+  const [open, setOpen] = useState(props.startOpen);
```

Comment:
issue: When clicking the "Children" element that uses this component I'm receiving the following error:
```
Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>. Add a <tbody>, <thead> or <tfoot> to your code to match the DOM tree generated by the browser. Error Component Stack
    at tr (<anonymous>)
    at table (<anonymous>)
    at div (<anonymous>)
    at div (<anonymous>)
    at DropDetails (DropDetails.js:5:1)
```

Reply:
Error resolved. Waiting review🙂